packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -329,7 +329,7 @@ export default class HostItem extends Component {
             <Col mediumSize={2} className={styles.itemExpanded} style={{ marginTop: '0.3rem' }}>
               <span>
                 {host.cpus.cores}x <Icon icon='cpu' /> &nbsp; {formatSizeShort(host.memory.size)} <Icon icon='memory' />{' '}
-                &nbsp; v{host.version.slice(0, 3)}
+                {host.productBrand} v{host.version.slice(0, 3)}
```

Comment:
The `slice` will break once it reaches 2-digit numbers.

Also, why not showing the full version?